MapboxDirections/MBRouteOptions.swift

```diff
@@ -339,9 +345,19 @@ open class RouteOptions: NSObject, NSSecureCoding {
      */
     open var includesExitRoundaboutManeuver = false
     
-    open var includeVoiceInstructions = false
+    /**
+     A Boolean value indicating whether the route includes an array of `VoiceInstructions`.
+     
+     The order of each `VoiceInstructions` in the array indicates the order of which each instruction should be given in.
+     */
+    open var includesVoiceInstructions = false
     
-    open var unitsForVoiceInstructions: VoiceUnit = .imperial
+    /**
+     The distance units used for the `includesVoiceInstructions`.
+     
+     `includesVoiceInstructions` must be true for this property to have any effect.
+     */
+    open var distanceMeasurementSystem: MeasurementSystem = .imperial
```

Comment:
The default should not be `imperial`. If we need to choose a default, it should be `metric`, the international standard. But I think it would be better to default to a dynamic value based on the `locale`, which would require #133 to be implemented first.